Tools/TestWebKitAPI/Tests/WebKitCocoa/IPCSerialization.mm

```diff
@@ -74,7 +74,9 @@ CFTypeRef valueAsCFType() const
     typedef std::variant<
         RetainPtr<CFStringRef>,
         RetainPtr<CFURLRef>,
-        RetainPtr<CFDataRef>
+        RetainPtr<CFDataRef>,
+        RetainPtr<CFBooleanRef>,
+        RetainPtr<CGColorRef>
```

Comment:
Nit: Would be nice if these were sorted.  Not required to land PR.

Reply:
(As you guessed) we're just adding them chronologically as we go.
I'll land this one as-is, and we can start sorting next time.